tests/jcond.s

```diff
@@ -0,0 +1,31 @@
+jmp #5                      ; keep pass/fail at fixed location
+print ^cond_jump_insts_ok   ; overall pass
+halt
+print ^cond_jump_insts_fail ; any fail
+halt
+sub $1 $2 #3                ; test all j insts that don't consider < case as match
```

Comment:
I didn't get this, your comment says "don't consider < case as match", but what is "< case"?
Below code are testing with "==", ">=" and ">", and test with -1, I don't think it match your comment.